app/code/Magento/Tax/view/frontend/web/js/view/checkout/cart/totals/tax.js

```diff
@@ -21,7 +21,7 @@ define([
          * @override
          */
         ifShowValue: function () {
-            if (this.getPureValue() === 0) {
+            if (this.getPureValue() == 0) {
```

Comment:
This change makes the logic less robust, we should avoid using non-strict comparison even though JS allows this.